### PR TITLE
[TRANSLATION] Improve Italian (it-IT) translation

### PR DESCRIPTION
**Pull request by Gabriele007xx**
## Purpose

I have edited some traslation files for the italian language, as the text looked robotic in the way senteces were written, or in some cases it used not-really-fitting synonymous or incorrect spelling. The edited strings are from varius areas of ReactOS. I hope this contribution makes italian language less strange to native speakers.

## Proposed changes

-  Improvement of italian traslation

### CODE REVIEW

File: dll/shellext/devcpux/lang/it-IT.rc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
```
Comment (binarymaster) [inline dll/shellext/devcpux/lang/it-IT.rc:1]:
```suggestion
/*
 * PROJECT:     ReactOS Processor Shell Extension
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Italian resource file
 * TRANSLATOR:  Copyright 2024 Gabriele Lo Re <your@email.com>
 */

LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
```
(make sure to set your contact email here)

**Review by binarymaster — COMMENTED** [review]
`dll/shellext/devcpux/lang/it-IT.rc` should be added to the main RC file.

**Comment by binarymaster** [comment]
Cc @reactos/lang-italian for review.

File: dll/cpl/main/lang/it-IT.rc [hunk 2]
```diff
@@ -50,56 +50,56 @@ END
...
+    PUSHBUTTON "&Elimina", IDC_BUTTON_DELETE_SCHEME, 125, 30, 40, 15
     CONTROL "", IDC_IMAGE_CURRENT_CURSOR, "Static", SS_ICON | SS_CENTERIMAGE | WS_VISIBLE, 190, 9, 45, 41, WS_EX_STATICEDGE
     LTEXT "&Personalizza", -1, 5, 60, 48, 10
     LISTBOX IDC_LISTBOX_CURSOR, 5, 70, 230, 120, WS_VISIBLE | WS_BORDER | WS_TABSTOP | WS_VSCROLL | LBS_OWNERDRAWFIXED | LBS_NOTIFY
-    CHECKBOX "&Abilita l'ombra del puntatore", IDC_CHECK_CURSOR_SHADOW, 5, 200, 108, 10
-    PUSHBUTTON "Usa predefinito", IDC_BUTTON_USE_DEFAULT_CURSOR, 120, 199, 60, 15
-    PUSHBUTTON "&Esplora", IDC_BUTTON_BROWSE_CURSOR, 185, 199, 50, 15
+    CHECKBOX "&Abilita ombra puntatore", IDC_CHECK_CURSOR_SHADOW, 5, 200, 108, 10
+    PUSHBUTTON "Predefinito", IDC_BUTTON_USE_DEFAULT_CURSOR, 120, 199, 60, 15
+    PUSHBUTTON "&Sfoglia....", IDC_BUTTON_BROWSE_CURSOR, 185, 199, 50, 15
```
Comment (JoachimHenze) [inline dll/cpl/main/lang/it-IT.rc:86]:
```suggestion
    PUSHBUTTON "&Sfoglia...", IDC_BUTTON_BROWSE_CURSOR, 185, 199, 50, 15
```

File: dll/shellext/devcpux/lang/it-IT.rc [hunk 3]
```diff
@@ -0,0 +1,48 @@
+/*
+ * PROJECT:     ReactOS Processor Shell Extension
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Italian resource file
+ * TRANSLATOR:  Copyright 2024 Gabriele Lo Re <your@email.com>
```
Comment (JoachimHenze) [inline dll/shellext/devcpux/lang/it-IT.rc:5]: `<your@email.com>` is a just template. You shall fill it with something real. Here and in other files where you have might done it like that.

File: dll/win32/browseui/lang/it-IT.rc [hunk 4]
```diff
@@ -150,7 +150,7 @@ BEGIN
     DEFPUSHBUTTON "Sea&rch", IDC_SEARCH_BUTTON, 80, 110, 40, 15, WS_GROUP | WS_TABSTOP | WS_VISIBLE
     PUSHBUTTON "&Stop", IDC_SEARCH_STOP_BUTTON, 80, 110, 40, 15, WS_GROUP | WS_TABSTOP
 
-    AUTOCHECKBOX "Search Hidden Files and Folders",
+    AUTOCHECKBOX "Cercca file e cartelle nascoste:",
```
Comment (JoachimHenze) [inline dll/win32/browseui/lang/it-IT.rc:153]: Unlikely that MS ends an autocheckbox with `:` Please prove that with a screenshot of italian MS Windows, or strip it.

File: dll/shellext/devcpux/lang/it-IT.rc (hunk 3 above)
Comment (Gabriele007xx) [reply dll/shellext/devcpux/lang/it-IT.rc:5]: yeah, @binarymaster added it as it was missing. Next time when I can I will commit it with my email.

File: dll/win32/browseui/lang/it-IT.rc (hunk 4 above)
Comment (Gabriele007xx) [reply dll/win32/browseui/lang/it-IT.rc:153]: Can you please request a change here? (gonna strip it as indeed it hasn't the ':' in the italian veesion and 'Cerca' has a typo, being 'Cercca'

**Comment by binarymaster** [comment]
> Next time when I can I will commit it with my email.

Alright, at the time being please uncheck `Keep my email addresses private` in [GitHub email settings](https://github.com/settings/emails).

This would allow merging with correct email address. Currently it uses a no-reply address:

image

**Comment by Gabriele007xx** [comment]
> Alright, at the time being please uncheck `Keep my email addresses private` in [GitHub email settings](https://github.com/settings/emails).

Did it, thanks for letting me know.

File: dll/shellext/devcpux/lang/it-IT.rc (hunk 3 above)
Comment (binarymaster) [reply dll/shellext/devcpux/lang/it-IT.rc:5]:
```suggestion
 * TRANSLATOR:  Copyright 2024 Gabriele Lo Re <gabrolr70@gmail.com>
```

**Comment by simonelombardo** [comment]
Hi @Gabriele007xx , thanks for the italian translations! :)

I have a couple of minor suggestions from my part to discuss about, Please feel free to give your opinion.

>"Un tema è uno sfondo più un set di suoni, icone, e altri elementi per personalizzare il computer in un clic.", IDC_STATIC, 5, 5, 235, 30

I would translate "set" with "insieme" or "combinazione" (since it was used in another translated item)

>     LTEXT "Se selezioni un tema che non sia ReactOS classico, sovrascriverà le seguenti impostazioni, eccetto in alcuni vecchi programmi.", IDC_ADVAPPEARANCE_DISCLAIMER, 7, 133, 232, 30, BS_MULTILINE

(it's a bit ambiguous  in the english form to me too) the subject is unclear for the usage of "sovrascriverà" - "le seguenti impostazioni verranno sovrascritte" or "Selezionare un tema..." makes it more related to the action.

>     LTEXT "Mantenere queste opzioni di visualizzazione?", IDC_STATIC, 7, 7, 219, 8
>     IDS_TIMEOUTTEXT "Ripristino vecchie opzioni in %d secondi."
>     LTEXT "Se selezioni un tema che non sia ReactOS classico, sovrascriverà le seguenti impostazioni, eccetto in alcuni vecchi 
programmi.", IDC_ADVAPPEARANCE_DISCLAIMER, 7, 133, 232, 30, BS_MULTILINE
...
"opzioni" e "impostazioni" are used for translating "impostazioni" in the translation files. Is there any criteria to have used one or the other word?

>     IDS_DESKTOP_ITEMS "Oggetti Desktop"
It's a translation of "items" sounds like a list, so "elementi" or "voci" could be more effective

>     IDS_OVERWRITE_TEXT "Il nome della combinazione dei cursori è già in uso. Volere sovrascrivere quello esistente?"
Two infinite verbs. "Volere" = "Si vuole"?

>     IDS_ASK_TERMINATE "Questa applicazione non risponde. Terminare il processo?"
it's a bit risky to interchange "process" with "application" since these terms have a technical meaning that are different, even though it's colloquially used to refer the same thing and it was not used in the english translation. 

>     LTEXT "Se ci sono problemi ad eseguire questo programma in ReactOS potete scegliere un sistema operativo da segnalare all'applicazione.\nPer favore notare questo è un Work-In-Progress.", IDC_EXPLANATION, 7, 7, 210, 31

Is there a reason for the usage of Work-In-Progress instead of "lavoro in corso" that's more familiar for non technical italian audience?

>     IDS_PARSE_ADDR_ERR_TEXT "ReactOS Explorer non può trovare '%1'. Controlla la digitazione e riprova."

"può trovare" sounds like that ReactOS never attempted to search the term. "non trova" is more assertive for the user.

**Comment by Gabriele007xx** [comment]
> I would translate "set" with "insieme" or "combinazione" (since it was used in another translated item)

I agree, 'insieme' it's literally the correct native traslation. Sometimes, due to habit, one keeps english words like it's part of their native language. Out of the two, I would pick 'insieme' since it's shorter and still effective.
> Is there a reason for the usage of Work-In-Progress instead of "lavoro in corso" that's more familiar for non technical italian audience?

Yeah. same reason above. Never heard this slang outside technical situations. Does an italian counter-part slag even exist?

>(it's a bit ambiguous in the english form to me too) the subject is unclear for the usage of "sovrascriverà" - "le seguenti impostazioni verranno sovrascritte" or "Selezionare un tema..." makes it more related to the action.

Thanks for the grammatical correction. I would change it to "le seguenti impostazioni verranno sovrascritte" then, just to keep the label short and avoid resizing.

>It's a translation of "items" sounds like a list, so "elementi" or "voci" could be more effective

Official Italian Windows calls them 'elementi' if I am not mistaken, so that would be the right.

> "opzioni" e "impostazioni" are used for translating "impostazioni" in the translation files. Is there any criteria to have used one or the other word?

No particular criteria. If we are going to make a difference, I think 'Impostazioni' would fit more the control panel, while 'opzioni' what's found in tabs after going in a particular section.

>Two infinite verbs. "Volere" = "Si vuole"?
Yeah. let's change it to 'Si vuole', as now that I think about it the current sounds too robotic.

>it's a bit risky to interchange "process" with "application" since these terms have a technical meaning that are different, even though it's colloquially used to refer the same thing and it was not used in the english translation.

I agree, let's change it to 'processo', as those two terms have a different technical meaning. If I am remembering correctly now, when the same dialog window pops out in Italian Windows, it's called 'processo'.

>"può trovare" sounds like that ReactOS never attempted to search the term. "non trova" is more assertive for the user.

Would 'non è in grado di trovare' fit better?

**Comment by simonelombardo** [comment]
> Would 'non è in grado di trovare' fit better?

Agreeing with this and all other points! :)

File: win32ss/user/winsrv/usersrv/lang/it-IT.rc [hunk 5]
```diff
@@ -34,16 +34,16 @@ END
...
 END
 
 STRINGTABLE
 BEGIN
-    IDS_SYSTEM_PROCESS          "System Process"
+    IDS_SYSTEM_PROCESS          "Processo di Sistema"
     IDS_UNKNOWN_SOFTWARE_EXCEPT "unknown software exception"
-    IDS_OK_TERMINATE_PROGRAM    "Click on OK to terminate the program."
-    IDS_CANCEL_DEBUG_PROGRAM    "Click on CANCEL to debug the program."
+    IDS_OK_TERMINATE_PROGRAM    "Premere Ok per terminare il programma."
```
Comment (binarymaster) [inline win32ss/user/winsrv/usersrv/lang/it-IT.rc:47]:
```suggestion
    IDS_OK_TERMINATE_PROGRAM    "Premere OK per terminare il programma."
```

**Comment by carlo-bramini** [comment]
Unfortunately, it looks like none of the mistakes signaled by me have been fixed.

**Comment by binarymaster** [comment]
@carlo-bramini I've posted a message in chat on that, just in case: https://chat.reactos.org/reactos/pl/m1xd657gbtr7dnshdetrzqe1gw

**Pull request merged**